docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3"
+
+services:
+  bnetd:
+    image: pvpgn-server:bnetd-mysql
+    restart: unless-stopped
+    ports:
+      - 6112:6112/udp
+      - 6112:6112
+    volumes:
+      - /your/config/dir:/usr/local/etc/pvpgn
```

Comment:
How about we change this to something more probable, let's say `/opt/pvpgn/etc` and `/opt/pvpgn/var`?